content/getting-started/sections/installing-atom.md

```diff
@@ -70,13 +70,32 @@ With Atom open, click on `File > Settings`, and then the `System` tab on the lef
 
 #### Installing Atom on Linux
 
-To install Atom on Linux, you can download a [Debian package](https://atom.io/download/deb) or [RPM package](https://atom.io/download/rpm) either from the [main Atom website](https://atom.io) or from the [Atom project releases page][releases]. These packages do not currently have auto-update features, so when you would like to upgrade to a new release of Atom, you will have to repeat this installation process.
+To install Atom on Linux, you can download a [Debian package](https://atom.io/download/deb) or [RPM package](https://atom.io/download/rpm) either from the [main Atom website](https://atom.io) or from the [Atom project releases page][releases]. If you'd like Atom to be updated when new releases are published, you can configure an official package repository for your distribution.
```

Comment:
Hmm I find it weird that we tell them they can download from the main site but now that the instructions are more towards installing using the package repository.  I wonder if we should remove the section about being able to download the builds or maybe have a separate section for manually installing.

Reply:
Yeah, I thought about that too.  Since I provide the alternative of installing the packages individually (with links to the download link in each distro section), I can remove these links from the top.  How about this wording instead:

> You can install Atom on Linux using your distribution's package manager by configuring it to use one of our official package repositories.  This will also allow you to update Atom when new releases are published.

Reply:
Sounds good!  Should we include a section about how to manually install too?  I guess we shouldn't since this is way better.

Reply:
The manual install bits are still there, check out these sections, just search for the word "Alternatively" and you'll see the manual steps for the 3 package managers